scripts/jenkins/ardana/ansible/roles/input_model_generator/tasks/main.yml

```diff
@@ -28,7 +28,7 @@
   when: "item ~ '_template' in scenario"
 
 - include_tasks: get_qe_bm_info.yml
-  when: is_physical_deploy
```

Comment:
This is not a bare variable. How did you come to want to change this?

https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html says:
"when clause, which contains a raw Jinja2 expression without double curly braces"
and gives this as an example with: "when: epic".

The lint does not check for when: https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/rules/UsingBareVariablesIsDeprecatedRule.py

Reply:
Interesting. I got a warning with Ansible 2.8. I'll double check that.

Reply:
The warning I am seeing is here: https://github.com/ansible/ansible/blob/61ff61cdf6a6cfb749d550a8ea781856dfda73a4/lib/ansible/playbook/conditional.py#L136

But that code is only in `Ansible-2.8.1` which is excluded since #3498 was merged. We can close this, unless we want to "future proof" the playbook in case we ever unpin Ansible.

Reply:
Thx. That means it is a bug in the ansible docs and this isn't set to a bool where you saw the warning.

Reply:
Though the ansible docs will be correct once the behaviour is deprecated.